app/src/main/java/com/xmartlabs/template/ui/common/BaseViewHolder.java

```diff
@@ -0,0 +1,43 @@
+package com.xmartlabs.template.ui.common;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+
+import butterknife.ButterKnife;
+
+/**
+ * A Base View Holder
+ * Setting, removing, adding, getting items count.
+ */
+public class BaseViewHolder extends RecyclerView.ViewHolder {
+
+  /**
+   * View holder default constructor.
```

Comment:
This doesn't add any information. Also, it isn't the "default constructor". Maybe explain what the param does